Ds 218

AshGDS opened the pull request:
Hi @matt-blair,

Would you be able to look at this branch and merge it if it's OK? 

It is mainly HTML and CSS changes in response to some design changes to the collection explorer.

I've put all the links to each page on the index page of the PHP application, so they should be easy to access.

This contains changes from these designs by SW. I have pasted them below. Thanks :+1:

![image](https://user-images.githubusercontent.com/8880610/125286807-1ff60500-e314-11eb-9dda-a795788dd140.png)
![image](https://user-images.githubusercontent.com/8880610/125286821-23898c00-e314-11eb-8452-05d54b31ccb1.png)
![image](https://user-images.githubusercontent.com/8880610/125286837-26847c80-e314-11eb-8125-0be0e3f2e790.png)

AshGDS committed:
Add collection explorer UI changes

These include:
- Adding a new index page with topic/time period selection
- Adding a prefix to the generic intro H1 which appears as a smaller
  block above the main H1 text
- Adding item count to the results options component
- Content updates

AshGDS committed:
Fix typo

AshGDS committed:
Fix logo BETA font and add header text

AshGDS committed:
Update promo cards & explorer card images

This includes major style changes to the promo cards, and minor image changes to the explorer cards.

AshGDS committed:
Change 'More ways to explore' promo card to green on time period page

AshGDS committed:
Increase heading text size and change results options colour

AshGDS committed:
Change results options text

AshGDS committed:
Improve heading text alignment

AshGDS committed:
Add new card image widths

AshGDS committed:
Add collection explorer links to homepage

AshGDS committed:
Remove unused colour

matt-blair commented on sass/includes/_results-options.scss:
```diff
@@ -1,10 +1,21 @@
 .results-options {
-  text-align: right;
+  background-color: $color__green;
+
```
Might be good to close the empty line here.

AshGDS replied:
Done :+1:

matt-blair commented on sass/includes/_header.scss:
```diff
@@ -1,12 +1,43 @@
 .tna-header {
+  padding: 1rem 3rem 0rem 3rem;
```
Might need to add a media query to reduce left and right padding to match the content padding in the main container on smaller screen sizes. Currently different alignments,

AshGDS replied:
Thanks @matt-blair - i've realised that my header was completely different to Simon's design, so I have fixed it by wrapping it in a bootstrap container.

matt-blair replied:
@AshTNA I think what you had before was right, just with a little less padding. We need the header to be full width rather than using a bootstrap container because the details page doesn't use container width for its main content area so there's an alignment issue. Making the header full width will mean it works across all 3 prototypes. I'd probably revert it back to what you had originally and then just change the padding-left and padding-right to 1rem rather than 3rem on mobile. That should do the job.

AshGDS replied:
Thanks @matt-blair - I see what you mean now, i've pushed the change. :+1:

matt-blair replied:
That's great. I think the bottom padding should probably change from 0rem to 1rem in the media query too as it causes a problem on mobile being quite tight against the generic-intro panel. Hopefully after that it'll be resolved.

AshGDS replied:
Thanks @matt-blair - I have updated this now too!

matt-blair commented on sass/includes/_generic_intro.scss:
```diff
@@ -18,5 +18,15 @@
+      font-size: 2rem;
+      line-height: 0.3em;
+      margin-top: 1em;
+    }
```
Line-height or bottom margin needs increasing on mobile as it looks a bit tight against the larger heading size below on smaller screens.

AshGDS replied:
I've added a line-height of `line-height: 1.3em;` on mobile :+1:

matt-blair replied:
Looks good!

matt-blair commented on includes/header.php:
```diff
@@ -1,6 +1,10 @@
+    <p class="tna-header__description">
+    <span class="tna-header__description-heading">Welcome to The National Archives</span><br/>
+    We are the guardians of over 1,000 years of iconic documents and UK government records.
+    <p>
```
More an editorial question/comment (not sure if it's been looked over by a content designer) but I don't think we need Welcome to The National Archives. It's a bit redundant.

AshGDS replied:
Thanks @matt-blair - I've been told that the heading is subject to change so there may be a different heading there so we can treat that as a placeholder.

matt-blair reviewed (changes requested):
Hi Ash, I've made a few single line comments which need resolving. Most are fairly straight-forward, one is more a question for Simon and the rest of the team about the strapline in the heading. (Not sure if the PR is the right place for it, but just thought it'd be worth checking before approving).

Matt.

AshGDS committed:
Changes in response to PR

Remove empty line from CSS, fix header alignment, and improve line-height on mobile generic intro heading

AshGDS committed:
Merge branch 'DS-218' of github.com:nationalarchives/ds-wagtail-frontend into DS-218

AshGDS committed:
Merge branch 'main' into DS-218

AshGDS committed:
Revert container on heading and improve mobile padding

AshGDS committed:
Fix closing p tag

AshGDS committed:
Change header padding on mobile
